content/projects/java-specific/introduction-to-spring-boot/view-template/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+content_type: project
+flavours:
+- java
+prerequisites:
+  hard:
+  - topics/java-specific/introduction-to-spring-boot/view-template
+  soft: []
+ready: true
+submission_type: repo
+tags:
+- spring-boot
+- rest-api
+- rest-templates
+title: Spring Boot - View Template with Thymeleaf and Upload Files
+---
+
+Java is mostly used for backend application but sometimes you might need to harnes javascript skill within java to run some front application that does something that mostly is backend heavy, this is what we are going to do here
+
+In this project we are going to upload a file from the frontend and process it in the backend
+
+#### Step 1
```

Comment:
Are you sure all the heading levels make sense? Step 1 is <h4> and step 2 is <h3>

Reply:
thank you, fixed